src/Assignment-3/assignment3.css

```diff
@@ -5,3 +5,80 @@
 width: 100%;
 height: 100%;
 }
+*{
+    margin:0;
+    padding:0;
+}
+body{
+    background-color: #f2f2f2;
+}
+.first{
+    margin-top:5vw;
+    margin-left: 5vw;
+    margin-right:5vw;
+    width:90vw;
+    
+    display:flex;
+    flex-wrap: wrap-reverse;
+    flex-direction: row;
+    height:75vh;
+}
+.content-first{
+    display:flex;
+    width:50%;
+    flex-direction: column;
+    justify-content:space-evenly;
+    
+}
+#render-wiki{
+    display:inline-block;
+    height:100%;
+    width:100%;
+}
+.iframe-parent{
+    width: 50%;
+}
+h1{
+    font-size: 3rem;
+    transform: translate();
+}
+button{
+    color:white;
+    border-radius: 0;
```

Comment:
remove this property, since it has no effect

Reply:
it does.. it removes default styling of button @chitha-srinath